HACKING.md

```diff
@@ -38,69 +38,37 @@ that require a cluster to properly test your code.
 
 In those situations, you will need to:
 
-1) build the NTO image you're interested in testing
+1) Build the NTO image you're interested in testing
 2) Push the image to an OpenShift cluster
 
-## Build NTO image
-
-To build an image that contains all NTO components run:
+Before continue with the next steps, please change the following environment variables in the **makefile**: 
 
 ```
 # Specify the registry
-REGISTRY=quay.io
+REGISTRY=registry_name, default will use quay.io
 # Specify the registry user
 ORG=username
-# Specify the tag
+# Specify the tag, default will use branch name
 TAG=tag
-
-make local-image IMAGE=${REGISTRY}/${ORG}/cluster-node-tuning-operator:${TAG}
 ```
 
+Verify the `IMAGE` variable is being constructed as you expect.
+
+# Build NTO image
+
+To build an image that contains all NTO components run:
+
+`make local-image`
```

Comment:
this works only if the `IMAGE` environment variable is `export`ed, right?

Reply:
Yes. The `IMAGE` is set in the makefile.

Reply:
Shell variable `export` will not help you.  You need to specifically set make variables on the command line.  We should probably be more specific here and say something like:

```
make local-image IMAGE=$IMAGE
```

This needs to be fixed in the docs.